src/test/simple_test.cpp

```diff
@@ -0,0 +1,70 @@
+
+#include "simple_test.hpp"
+#include <gtest/gtest.h>
+
+using ppT = libff::default_ec_pp;
+using FieldT = libff::Fr<ppT>;
+using namespace libsnark;
+
+namespace
+{
+
+TEST(SimpleTests, SimpleCircuitProof)
```

Comment:
I'd stick to `TEST(SimpleTests, SimpleCircuitProof) {` and move the bracket up for consistency with other files.

Reply:
The bracket on the next line follows the newly adopted style conventions.